Backpack/HorizontalNavigation/Classes/BPKHorizontalNavigation.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * `BPKHorizontalNavigation` is a subclass of `UIStackView`.
+ * dequeueReusableCell method can be overridden to create cutom looking cells. By default it will just return a new
+ * instance of BPKHorizontalNavigationCell
+ */
+@class BPKHorizontalNavigationCell;
+@class BPKHorizontalNavigationDelegate;
+NS_SWIFT_NAME(HorizontalNavigation) IB_DESIGNABLE @interface BPKHorizontalNavigation : UIStackView
```

Comment:
I kind of feel like this should be a `UIControl` subclass like `UISegmentedControl` is. At the very least making it a `UIStackView` subclass feels like an abstraction leak so if it isn't `UIControl` it should be `UIView`